index.js

```diff
@@ -349,7 +349,21 @@ class AntiSpamClient extends EventEmitter {
 		}
 		this.cache.messages = this.cache.messages.filter((u) => u.authorID !== message.author.id)
 		this.cache.mutedUsers.push(message.author.id)
-		const role = message.guild.roles.cache.find(role => role.name === this.options.muteRoleName)
+		const role = message.guild.roles.cache.find(role => role.name === this.options.muteRole) || message.guild.roles.cache.get(this.options.muteRole)
+		if(!role){
+			if (this.options.verbose) {
+				console.log(`DAntiSpam (kickUser#userNotMutable): ${message.author.tag} (ID: ${message.author.id}) could not be muted, improper permissions or the mute role couldn't be found.`)
+			}
+			if (this.options.errorMessages) {
+				await message.channel
+					.send(this.format(this.options.muteErrorMessage, message))
+					.catch((e) => {
+						if (this.options.verbose) {
+							console.log(`DAntiSpam (muteUser#muteRoleNotFound): ${e.message}`)
+						}
+					})
+			}
```

Comment:
Here you should add `return false` so the rest of the code won't be run.

Reply:
Added

Reply:
Not here. Above, so it returns if the role does not exist.

Reply:
It is on the if(!role)

Reply:
![image](https://user-images.githubusercontent.com/48729921/117183150-7e17e080-add7-11eb-822c-9acbeb522600.png)
look

Reply:
nvm fixed it in the last commit.